insights/parsers/image_builder_facts.py

```diff
@@ -0,0 +1,25 @@
+"""
+ImageBuilderFacts - files ``/etc/rhsm/facts/osbuild.facts``
+=========================================================
```

Comment:
For insights-core doc build, the length of this line shall be align to the line above.